lib/csrf_token_verifier.rb

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 
 # Provides a way to check a CSRF token outside of a controller
-class CSRFTokenVerifier
-  class InvalidCSRFToken < StandardError; end
+class CsrfTokenVerifier
```

Comment:
I think it will be possible to add an acronym for `CSRF`?

Reply:
Sorry I looked through the discussion but wasn't able to tell what the conclusion was.

Reply:
To avoid having exceptions in `Rails.autoloaders.main.ignore` and inflections if we can just follow Rails naming conventions (that’s what I understood at least :sweat_smile:)

Reply:
OK I think we should just fix this. Breaking constant during a Rails 7 upgrade is not something we want to do.